irodsbackend/static/irodsbackend/js/irodsbackend.js

```diff
@@ -247,18 +270,24 @@ $(document).ready(function() {
                     }
                     objLink += obj['name'] + '</a>';
 
-                    var colSpan = '1';
-                    var icon = 'mdi:file-document-outline';
-                    var toolTip = 'File';
+                    var colSpan = obj['type'] === 'coll' ? '1' : '4';
+                    var icon = obj['type'] === 'coll' ? 'mdi:folder-open' : 'mdi:file-document-outline';
+                    var toolTip = obj['type'] === 'coll' ? 'Collection' : 'File';
+
+                    var copyButton = '';
                     if (obj['type'] === 'coll') {
-                        colSpan = '4';
-                        icon = 'mdi:folder-open';
-                        toolTip = 'Collection';
+                        copyButton = '<button class="btn btn-secondary sodar-list-btn pull-right" ' +
+                            'id="sodar-irods-copy-btn-' + obj['name'] + '" ' +
+                            'title="Copy iRODS path into clipboard" data-tooltip="tooltip" ' +
+                            'data-placement="top" onclick="copy_path(\'' + obj['path'] + '\', \'sodar-irods-copy-btn-' + obj['name'] + '\')">' +
```

Comment:
While we don't enforce a specific line length for JQuery files at the moment, super long lines like this one should be broken down.